books/management/commands/addepub.py

```diff
@@ -249,7 +249,7 @@ def process_epub(self, filename, use_symlink=False):
                                 .split(',')
                             for tag in subject_split:
                                 if tag is not ' ':
-                                    tag = tag.encode("utf-8").lower().strip()
+                                    tag = tag.lower().strip()
```

Comment:
I'd be very interested in checking the results on your library! If there are indeed many pure-ascii tags, we might end up nesting some `try..catch` blocks:
* try to add the tag without changing the encoding first (same as it is now)
* if fails, try to add the tag forcing utf-8 (same as before)
* if not, ignore the tag

I'm not sure if the specifications mention something about the encoding as well?